### DISCUSSION TITLE
고하나 - 교육/공공시설 - 대학 도서관 열람석 사석화 발생 원인 파악

### DISCUSSION THREAD

**Discussion by ko9612** [dinnercoding/wanted-pre-onboarding-challenge-data#103] [2023년 11월 프리온보딩 데이터 챌린지]
# 도메인을 선택한 이유
- 기존에, 대학 도서관 열람실의 '사석화' 문제 해결을 목적으로, 통합 좌석 관리 시스템 개발 프로젝트를 진행한 적이 있습니다. 
기존 기획안에 객관적 지표를 추가하여 설득력을 높힘으로써, 기획안을 디벨롭하기 위해 해당 도메인을 선택했습니다.

`사석화: 공용 공간에 개인 물품을 놓아 자리를 맡아둔 뒤 장시간 자리를 비워 개인 자리처럼 이용하는 것`

  
# 구체적인 문제 정의
- 대학 도서관 열람실에서 발생하는 사석화 문제에 대한 정확한 원인을 파악한 후, 개선 방안을 모색함으로써 대학 도서관의 좌석 이용 효율을 향상시키고자 합니다.
- 특히, 현재의 좌석 수가 재학생들을 충분히 수용하고 있는지 여부를 확인하여 사석화 문제의 근본적인 원인을 밝히기 위해 전국 대학 도서관들의 열람석 수와 재학생 수에 대한 관계를 분석하고자 합니다.
- 대학도서관의 좌석규정에 따르면, 열람실에는 학생정원의 20%이상을 수용할 수 있는 좌석이 갖춰져야 합니다. 
즉, 좌석 당 학생 수는 5명 이하여야 하며, 전국 대학 도서관들을 대상으로 해당 규정에 대한 미준수 비율을 파악하고자 합니다.

# 데이터 업로드
- [한국교육학술정보원_대학 시설 및 설비(출처: 공공데이터포털)](https://www.data.go.kr/data/15071928/fileData.do#tab-layer-file)
- 필요한 데이터: 2020년(최근)기준 전국 대학 수, 재학생 수, 총 열람석 수

**Comment by ghost** [comment]
하나 님! 기존에 진행했던 프로젝트에 객관적 지표를 추가한다는 실용적인 목적이라니.. 제가 다 열정이 생기네요.
이번 기회에 객관적 지표를 잘 세워가시면 좋겠네요 ㅎㅎ
5 whys를 잘 실천해주시는 모습 너무 좋습니다~!!
